LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/scripting.html

```diff
@@ -23,4 +23,10 @@
   let booleanContext = window.matchMedia("(scripting)");
   assert_true(booleanContext.matches);
 }, "Check that scripting currently evaluates to true in the boolean context");
+
+test(() => {
+  let match_initialonly = window.matchMedia("(scripting: initial-only)");
+  assert_false(match_initialonly.matches);
+}, "Check that 'initial-only' never matches");
+
```

Comment:
We should probably not add this subtest as Luke said. I would add a non WPT layout test like this one: https://searchfox.org/wubkat/search?q=LayoutTests%2Ffast%2Fcss%2Fmedia-query-overflow-block-paged-print&path=&case=false&regexp=false

Test that `scripting: initial-only` does not match when printing.

Reply:
> We should probably not add this subtest as Luke said. I would add a non WPT layout test like this one: [searchfox.org/wubkat/search?q=LayoutTests%2Ffast%2Fcss%2Fmedia-query-overflow-block-paged-print&path=&case=false®exp=false](https://searchfox.org/wubkat/search?q=LayoutTests%2Ffast%2Fcss%2Fmedia-query-overflow-block-paged-print&path=&case=false&regexp=false)
> 
> Test that `scripting: initial-only` does not match when printing.

Thanks — made it so. On trunk the added test fails as expected, but with the patch applied, passes as expected.